templates/injector-deployment.yaml

```diff
@@ -104,6 +104,10 @@ spec:
               value: "{{ .Values.injector.agentDefaults.memRequest }}"
             - name: AGENT_INJECT_MEM_LIMIT
               value: "{{ .Values.injector.agentDefaults.memLimit }}"
+            - name: AGENT_INJECT_EPHEMERAL_REQUEST
+              value: "{{ .Values.injector.agentDefaults.ephemeralRequest }}"
+            - name: AGENT_INJECT_EPHEMERAL_LIMIT
+              value: "{{ .Values.injector.agentDefaults.ephemeralLimit }}"
```

Comment:
I think we'd need to only conditionally set these env vars if the helm values have been specified, and default the values to empty strings. See https://github.com/hashicorp/vault-k8s/pull/386 and its related comments for full details, but TL;DR this setting can affect the whole pod's ephemeral storage instead of just the injected container's if it's not set for the other containers.